morse_app/settings.py

```diff
@@ -39,8 +39,39 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'django.contrib.sites',
+
+    # 3rd party
+    'rest_framework',
+    'rest_framework.authtoken',
+    'dj_rest_auth',
+    'dj_rest_auth.registration',
+    'allauth',
+    'allauth.account',
+    'allauth.socialaccount',
+
+    # local
+    'users',
 ]
 
+# django-allauth config
+SITE_ID = 1
+ACCOUNT_EMAIL_REQUIRED = True
+ACCOUNT_USERNAME_REQUIRED = True
```

Comment:
Do these two `_REQUIRED` variables need to be here? Seems like they are only a concern of the database models

Reply:
django-allauth config needs to be in the settings of Django App